src/filesystem/unix/SDL_sysfilesystem.c

```diff
@@ -177,8 +177,39 @@ SDL_GetBasePath(void)
         }
 
         if (exe) {
-            if (realpath(exe, realpathbuf) != NULL) {
-                retval = realpathbuf;
+
```

Comment:
You still add unwanted newlines, but whatever. Maintainers should review first.

Reply:
How am i supposed to know what a "wanted new line" is? There's inconsistent usage of new lines all over the entire source tree.